spaceship/lib/spaceship/connect_api/testflight/testflight.rb

```diff
@@ -251,6 +251,12 @@ def get_builds_for_beta_group(group_id: nil)
           test_flight_request_client.get("betaGroups/#{group_id}/builds")
         end
 
+        def get_build_ids_for_beta_group(group_id: nil)
+          raise "group_id is nil" if group_id.nil?
```

Comment:
```suggestion
        def get_builds_for_beta_group(group_id:)
```

Reply:
hmm the thing is there's a separate method that fetches builds on beta groups. this one is specifically fetching just build ids. see the method just about this one 😁

Reply:
but, the required arg makes sense!!

Reply:
Ha! To be fair... my eyes were dilated when I reviewed this PR so I was 100% not seeing clearly 😛